shell/platform/windows/win32_window.cc

```diff
@@ -129,7 +129,9 @@ Win32Window::MessageHandler(HWND hwnd,
         current_height_ = height;
         window->HandleResize(width, height);
         break;
-
+      case WM_FONTCHANGE:
```

Comment:
Does this ever happen? IIRC the docs said that this is sent to top-level windows, which this isn't.

If we can get this here, then I'm not sure why we need runner-level changes and embedding API changes.

Reply:
I should be able to remove this. The only way this might be needed is application sends targeted windows message, but that probably won't happen in real life

Reply:
Per the latest discussion in the other PR, I think what we actually want to do is keep this, remove the embedding-level APIs, and forward the WM_FONTCHANGE message down in the top-level window implementation.